x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer_grid_in_discover_trial.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { TestData } from './types';
+
+const SHOW_FIELD_STATISTICS = 'discover:showFieldStatistics';
+import { farequoteDataViewTestData } from './index_test_data';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const PageObjects = getPageObjects(['common', 'discover', 'timePicker', 'settings']);
+  const ml = getService('ml');
+  const retry = getService('retry');
+  const dataViews = getService('dataViews');
+
+  const startTime = 'Jan 1, 2016 @ 00:00:00.000';
+  const endTime = 'Nov 1, 2020 @ 00:00:00.000';
+
+  function runTestsWhenDisabled(testData: TestData) {
+    it('should show view mode toggle but not Field stats tab', async function () {
+      await PageObjects.common.navigateToApp('discover');
+      if (testData.isSavedSearch) {
+        await retry.tryForTime(2 * 1000, async () => {
+          await PageObjects.discover.loadSavedSearch(testData.sourceIndexOrSavedSearch);
+        });
+      } else {
+        await dataViews.switchToAndValidate(testData.sourceIndexOrSavedSearch);
+      }
+
+      await PageObjects.timePicker.setAbsoluteRange(startTime, endTime);
+
+      await PageObjects.discover.assertViewModeToggleExists();
+      await PageObjects.discover.assertFieldStatsTableNotExists();
+    });
+  }
+
+  describe('field statistics in Discover', function () {
```

Comment:
We should add the `trial` context for this test suite name.

Reply:
Updated in https://github.com/elastic/kibana/pull/178916/commits/a7ddc2a0776a91b1009bca116a130e114cda23c0